pkg/apis/extensions/v1beta1/conversion_generated.go

```diff
@@ -3721,7 +3721,9 @@ func autoConvert_v1beta1_DaemonSetSpec_To_extensions_DaemonSetSpec(in *DaemonSet
 	if err := Convert_v1beta1_DaemonSetUpdateStrategy_To_extensions_DaemonSetUpdateStrategy(&in.UpdateStrategy, &out.UpdateStrategy, s); err != nil {
 		return err
 	}
-	// in.UniqueLabelKey has no peer in out
+	if err := api.Convert_string_ref_To_string(&in.UniqueLabelKey, &out.UniqueLabelKey, s); err != nil {
```

Comment:
It looks like roundTripTest was not running on these types. I think this should have caused it to fail...